src/object.d

```diff
@@ -3257,8 +3252,6 @@ private size_t getArrayHash(in TypeInfo element, in void* ptr, in size_t count)
 }
 
 
-// @@@BUG5835@@@ tests:
-
```

Comment:
> it's a link back to the impetus to create it

This is not normal practice - if it were, the source code would be packed with them. There needs to be a special reason for such a link, and you agreed there isn't anything special here. It's the same as removing dead code - the history is what github is for.

>  this unit test is confusing without the context of the comment.

The bugzilla entry is a meandering thread. If you want to suggest a proper comment, I'd be happy to put it in. The bugzilla comment is not helpful.

> why are the asserts marked  //fails  ?

I don't know. It's not in the scope of this PR.